ext/zlib/zlib.stub.php

```diff
@@ -23,93 +23,93 @@ function readgzfile(string $filename, int $use_include_path = 0): int|false {}
 
 function zlib_encode(string $data, int $encoding, int $level = -1): string|false {}
 
-function zlib_decode(string $data, int $max_decoded_len = 0): string|false {}
+function zlib_decode(string $data, int $length = 0): string|false {}
 
 function gzdeflate(string $data, int $level = -1, int $encoding = ZLIB_ENCODING_RAW): string|false {}
 
 function gzencode(string $data, int $level = -1, int $encoding = ZLIB_ENCODING_GZIP): string|false {}
 
 function gzcompress(string $data, int $level = -1, int $encoding = ZLIB_ENCODING_DEFLATE): string|false {}
 
-function gzinflate(string $data, int $max_decoded_len = 0): string|false {}
+function gzinflate(string $data, int $length = 0): string|false {}
 
-function gzdecode(string $data, int $max_decoded_len = 0): string|false {}
+function gzdecode(string $data, int $length = 0): string|false {}
 
-function gzuncompress(string $data, int $max_decoded_len = 0): string|false {}
+function gzuncompress(string $data, int $length = 0): string|false {}
 
 /**
- * @param resource $fp
+ * @param resource $stream
  * @alias fwrite
  */
-function gzwrite($fp, string $str, ?int $length = null): int|false {}
+function gzwrite($stream, string $data, ?int $length = null): int|false {}
```

Comment:
As this is an alias of fwrite() we should probably use the same argument names in both:

```
/** @param resource $stream */
function fwrite($stream, string $string, ?int $max_length = null): int|false {}
```

Reply:
Oh, even though I check `fwrite`, didn't notice it uses the `$max_length` name. I'll synchronize the signatures then